package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "Todolist project from Odin-Project, the web app keep track of user's todo-list.",
+  "private": true,
+  "devDependencies": {
+    "webpack": "^4.39.2",
+    "webpack-cli": "^3.3.7",
+    "webpack-dev-server": "^3.8.0",
```

Comment:
We aren't actually using `webpack-dev-server` and `webpack-merge`. We can either make use of them or just remove them entirely from our project's dependencies.